README.md

```diff
@@ -83,6 +83,43 @@ non-zero length value.  Logs are sent to syslog and standard error by default.
 | SYSLOG_ERROR_PRIORITY | syslog.error     | `logger -p <value>`           |
 | VARDIR                | /var/lib/startup | Durable alternative to TMPDIR |
 
+# Automated Tests
+
+Automated tests are run inside of a docker image.  The initial tests are modeled
+as specification tests validating expected behavior.  [bats][bats] is the test
```

Comment:
Tiny nitpick: we don't define the bats link corresponding to this reference.

Reply:
Fixed in a3cea47...7e56437 HEAD -> test_structure (forced update)

Added link to https://github.com/sstephenson/bats